### DISCUSSION TITLE
编译4.5版本遇到问题求助

### DISCUSSION THREAD

**Discussion by 12wlj** [NCAR/hrldas#217] [Q&A]
您好，我要用到4.5版本的noah-mp模型，在尝试编译hrldas-hrldas_v4.5_master和hrldas-release-v4.5-WRF版本时候，最后编译显示生成了hrldas.exe，但是没有生成namelist.hrldas，这是什么原因呢？有没有解决方法，是makefile的问题吗？
下面是我的user_build_options
#=============================================================================================
#  Options for Linux with gfortran (not fully tested)
#=============================================================================================

 COMPILERF90    =       /usr/bin/gfortran
 FREESOURCE     =       -ffree-form  -ffree-line-length-none
 F90FLAGS       =       -g -fconvert=big-endian -fbounds-check -fno-range-check #-fno-underscoring
 MODFLAG        =       -I
 LDFLAGS        =       
 CPP            =       /usr/bin/cpp
 CPPFLAGS       =       -P -traditional -D_GFORTRAN_ # -D_HRLDAS_URBAN_
 LIBS           =       
 LIBJASPER      =       -L/home/zhanghd/Build_WRF_2/LIBRARIES_2/grib2/lib -ljasper -ljpeg
 INCJASPER      =       -I/home/zhanghd/Build_WRF_2/LIBRARIES_2/grib2/include/jasper
 NETCDFMOD      =      -I/home/zhanghd/Build_WRF_2/LIBRARIES_2/netcdf/include
 NETCDFLIB      =       -L/home/zhanghd/Build_WRF_2/LIBRARIES_2/netcdf/include/lib -lnetcdf -lnetcdff
 BZIP2          =       NO
 BZIP2_LIB      =       -L/usr/bin/bzip2 -lbz2
 BZIP2_INCLUDE  =       -I/bin
 RM             =       rm -f
 CC             =       /usr/bin/gcc

**Comment by cenlinhe** [comment]
Hi, there will be no namelist.hrldas generated after compilation. The namelist.hrldas for different examples are under /run/examples/ folder. For example: https://github.com/NCAR/hrldas/tree/hrldas_v4.5_develop/hrldas/run/examples
You can also use README.namelist under /run/ folder to see the meaning of each namelist entry and add/remove certain namelist entry.
